src/states/node.rs

```diff
@@ -871,8 +871,8 @@ impl Node {
                                                      src_name,
                                                      dst)
             }
-            (CandidateApproval { candidate_id, client_auth, sections }, Section(_), Section(_)) => {
-                self.handle_candidate_approval(candidate_id, client_auth, sections, outbox)
+            (CandidateApproval { candidate_id, client_auth, .. }, Section(_), Section(_)) => {
+                self.handle_candidate_approval(candidate_id, client_auth, outbox)
```

Comment:
You should also remove `sections` from `CandidateApproval` and `PeerManager::verified_candidate_info`. Doesn't need to land in this PR necessarily.